src/pages/MyProfile.jsx

```diff
@@ -1,15 +1,240 @@
+    const [data, setData] = useState({
+        newPass : '',
+        confirmPass : '',
+        token : tkn
+    });
+
+    const togglePass = () => {
+        setShowPass(prevState => !prevState);
+    }
+
+    const toggleConfirmPass = () => {
+        setShowConfirmPass(prevState => !prevState);
+    }
+    const handleChange = (e) => {
+        const { name, value } = e.target;
+        setData({...data, [name] : value});
+        e.target.focus();
+        console.log("Name: " + name + " , value: " + value);
+    }
+    const handleSubmit = async (e) => {
+        e.preventDefault();
+        try {
+            await api.post("/auth/reset/password", data);
+            navigate("/login");
+
+        } catch (err) {
+            toast("An error occured.");
+        }
+
+    }
+
+    const [open, setOpen] = useState(false);
+    const [openInvalidate, setOpenInvalidate] = useState(false);
+    const contentStyle = { borderRadius:'10px', width: '40%'};
+
     return(
         <PageLayout>
             <Header />
+            <span className="text-sky-700 font-bold text-3xl justify-self-center self-center ">Moj profil</span>
+
+            <PopupJS open={openInvalidate} closeOnDocumentClick={false} onClose={() => setOpenInvalidate(false)} modal {...{contentStyle}}>
+                <div className="flex justify-center my-10">
+                    <div className="flex-1 items-center rounded-bl-[10px] rounded-br-[10px] flex flex-col">
+                        <label className="font-bold text-sky-600 text-2xl mt-[30px] text-center">Želite li deaktivirati račun?</label>
+
+                        <div className="flex justify-between mt-[45px]">
+                            <button className="bg-red-400 text-white pl-9 pr-9 pt-1 pb-1 rounded-[5px] mr-7" onClick={() => { setOpenInvalidate(false); }}>Ne</button>
+
+                            <button className="bg-green-300 text-white pl-9 pr-9 pt-1 pb-1 rounded-[5px] ml-7" onClick={() => { invalidatePatient().then(r => setOpenInvalidate(false)); }}>Da</button>
+                        </div>
+                    </div>
+                </div>
+            </PopupJS>
+
+            <PopupJS open={open} closeOnDocumentClick={false} onClose={() => setOpen(false)} modal {...{contentStyle}}>
+                <div className="flex justify-center relative">
+                    <form className="flex-1 rounded-bl-[10px] rounded-br-[10px] flex flex-col items-center">
+                        <label className="font-bold text-sky-600 text-lg mt-[30px]">Nova lozinka:</label>
+                        <div className="relative">
+                            <input type={showPass ? "text" : "password"} name="newPass" id="newPass"
+                                   value={data.newPass}
+                                   onChange={handleChange}
+                                   className="w-[300px] h-[40px] bg-sky-200 opacity-50 rounded-[5px] p-2"/>

```

Comment:
This will cause issue. Especialy this large size 300px. Use full or numbers provided by Tailwind.